pkg/model/config/config.go

```diff
@@ -104,6 +87,19 @@ func (config *Config) AddResource(gvk GVK) bool {
 	return true
 }
 
+// HasGroup returns true if group is already tracked
+func (config Config) HasGroup(group string) bool {
+	// Return true if the target group is found in the tracked resources
```

Comment:
```suggestion
	// Return true if the target group is found in the tracked resources
	
	if  len(config.Resources) < 1 {
	      return false
	}
```

Reply:
That check is useless, if `len(config.Resources) == 0` then loop will finish inmediately, we do not need to check that it has any item first.